filebeat/input/v2/input-cursor/manager.go

```diff
@@ -154,7 +154,7 @@ func (cim *InputManager) Create(config *conf.C) (v2.Input, error) {
 
 	settings := struct {
 		ID           string        `config:"id"`
-		CleanTimeout time.Duration `config:"clean_timeout"`
+		CleanTimeout time.Duration `config:"clean_inactive"`
```

Comment:
Same here, we should rename `CleanTimeout` -> `CleanInactive`

Reply:
Fixed by 2f7d96a02a